docs/superpowers/plans/2026-05-31-imgproxy-option-grammar-simplification.md

```
@@ -0,0 +1,344 @@
+
+**Files:**
+- Modify: `lib/image_pipe/parser/imgproxy/option_grammar.ex` (stash working-tree edit)
+
+- [ ] **Step 1: Confirm the uncommitted refactor is present and is the only uncommitted change**
+
+Run:
+```bash
+cd /Users/hlindset/src/image_plug/.claude/worktrees/tender-diffie-3e2dbf
+git status --porcelain
+```
+Expected: exactly one line — ` M lib/image_pipe/parser/imgproxy/option_grammar.ex`. If there are other modified files, STOP and reconcile (this plan assumes the spike is the sole working-tree change). If that line is absent, the spike was already committed or lost — STOP and reconcile against the Appendix before continuing.
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)